test/v2/controllers/V2DeparturesControllerSpec.scala

```diff
@@ -791,61 +791,77 @@ class V2DeparturesControllerSpec
 
   }
 
-  "for retrieving a single message" - {
+  Seq(VersionedRouting.VERSION_2_ACCEPT_HEADER_VALUE, VersionedRouting.VERSION_2_ACCEPT_HEADER_VALUE_JSON_XML).foreach {
+    acceptHeaderValue =>
```

Comment:
You're only testing the accepted header, there are no checks to confirm the 406 behaviour